src/layouts/Docs.tsx

```diff
@@ -206,6 +204,7 @@ type DocsLayoutProps = Pick<
   | "lastUpdatedDate"
   | "crowdinContributors"
   | "contentNotTranslated"
+  | "gitContributors"
```

Comment:
Note that `gitContributors` and `creowdinContributors` will be deprecated in #12969 in favor of a single list of `contributors`